### PR TITLE
SDKS-3186: Moving iOS and Android samples and re-structuring the repo

### PR DESCRIPTION
**Pull request by george-bafaloukas-forgerock**
SDKS-3186: Moving iOS and Android samples and re-structuring the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,39 +7,33 @@
...
 ## Samples
 
-ForgeRock provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of ForgeRock and are not officially supported.
+Ping provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-To try out the ForgeRock JavaScript SDK please look at one of our samples:
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
 
-- [**Embedded login - `embedded-login`**](./embedded-login/README.md)
+To try out the Ping JavaScript SDK please look at one of our samples under [javascript](./javascript/)
```
Comment (forgerock-chris) [inline README.md:14]: I'm not sure about the naming in these three sentences, it's possibly also a bit wordy/repetitive.
Perhaps something like:
`We provide samples of integrating with Advanced Identity Cloud and PingAM for the following platforms:
 - Android
 - iOS
 - JavaScript`

File: README.md [hunk 2]
```diff
@@ -7,39 +7,33 @@
...
-  - Todo application that involves authentication and authorization of a user to post Todos with the `@forgerock/javascript-sdk` in Angular.
-    The main branch includes many flavors of callbacks, including WebAuthN, embedded login, and social login.
-
-<!------------------------------------------------------------------------------------------------------------------------------------>
-<!-- DOCS - Link off to the AM-centric documentation at sdks.forgerock.com. -->
+- Samples coming soon
 
 ## Documentation
 
 Documentation for the SDKs is provided on [ForgeRock Backstage](https://backstage.forgerock.com/docs/sdks/latest/index.html), and includes topics such as:
```
Comment (forgerock-chris) [inline README.md:26]:
```suggestion
Detailed [documentation](https://docs.pingidentity.com/sdks/latest/sdks/index.html) is provided, and includes topics such as:
```

File: README.md [hunk 3]
```diff
@@ -7,39 +7,33 @@
...
 
 - Introducing the SDK Features
-- Preparing your server for use with the SDKS
+- Preparing your server for use with the SDKs
 - API Reference documentation
 
 ## Requirements
 
+Javascript
```
Comment (forgerock-chris) [inline README.md:34]: I tend to use "JavaScript" everywhere.

File: android/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  <hr/>
+</p>
+
+## Samples
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping Android SDK please look at one of our samples:
```
Comment (forgerock-chris) [inline android/README.md:14]: Let's write around places where we say "the Ping Android SDK", as I don't think that's an official product name, and might clash with the hundreds of other SDKs we have ;)

Perhaps something like:
The following Android samples are available in this repo:

File: android/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+- [**Java Authenticator - `/java-authenticator`**](./java-authenticator/)
+  
+  - This Authenticator sample app supports registration of multiple accounts and multiple different authentication methods in each account, such as push notifications and one-time passwords.
+
+- [**Java UI prototype - `/java-ui-prototype`**](./java-ui-prototype/)
```
Comment (forgerock-chris) [inline android/README.md:20]: Thanks for renaming these, very tidy 👍

File: android/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  
+  - An example Android project making full use of the Ping SDK with a prototyping UI. Allows to explore the SDK uses in detail with example calls for running Journeys, getting OAuth2.0 tokens, User Info and more.
```
Comment (forgerock-chris) [inline android/README.md:22]: Let's avoid "Ping SDK" and just go with "SDK" to avoid any renames/clashes.

File: android/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  (https://backstage.forgerock.com/docs/sdks/latest/sdks/tutorials/android/index.html)
+
+
+### Integrate with PingOne DaVinci:
+
+- Samples coming soon
+
+## Requirements
+
+Android
```
Comment (forgerock-chris) [inline android/README.md:45]: Can probably delete "Android", we're in the Android readme already.

File: android/java-authenticator/README.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with MFA capabilities on PingOne Advanced Identity Cloud / PingAM::
+
+- This Authenticator sample app supports registration of multiple accounts and multiple different authentication methods in each account, such as push notifications and one-time passwords.
+
+### Getting started
+
+To try out the ForgeRock Android SDK sample, perform these steps:
```
Comment (forgerock-chris) [inline android/java-authenticator/README.md:18]: ~ForgeRock~

**Comment by forgerock-chris** [comment]
General search & replace: let's update to point to the Ping Identity docs site, we're redirecting from BackStage next week as I understand it.

Globally replace:
`https://backstage.forgerock.com/docs/`
with:
`https://docs.pingidentity.com/`

File: javascript/central-login-pingone/README.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+To configure this sample, follow the steps in [Getting Started](#getting-started).
+
+Then, run the sample app as follows:
+```
+npm run start:central-login
+``` 
+
+### Instructions
+
+* [ForgeRock server](#forgerock-server)
```
Comment (forgerock-chris) [inline javascript/central-login-pingone/README.md:20]: If this one applies to both PingOne and PingAM/AIC, did we decide to use "OIDC" as the "server" name?
I can't quite remember what convention we leaned towards.

**Review by forgerock-chris — APPROVED** [review]

File: android/README.md (hunk 6 above)
Comment (george-bafaloukas-forgerock) [reply android/README.md:22]: See comment above

File: README.md (hunk 3 above)
Comment (george-bafaloukas-forgerock) [reply README.md:34]: Fixed

File: javascript/README.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+  - Todo application that involves authentication and authorization of a user to post Todos with the `@forgerock/javascript-sdk` in ReactJS.
+    The main branch includes many flavors of callbacks, including WebAuthN, embedded login, and social login.
+
+- [**Angular Todo - `angular-todo`**](./javascript/angular-todo/README.md)
+  
+  - Todo application that involves authentication and authorization of a user to post Todos with the `@forgerock/javascript-sdk` in Angular.
+    The main branch includes many flavors of callbacks, including WebAuthN, embedded login, and social login.
+
+
+### Integrate with PingOne DaVinci:
```
[the remark this answers is not in the document]
Comment (george-bafaloukas-forgerock) [reply javascript/README.md:35]: I changed that to "Integrate with PingOne" 

More generic for now

File: javascript/central-login-pingone/README.md (hunk 9 above)
Comment (george-bafaloukas-forgerock) [reply javascript/central-login-pingone/README.md:20]: I have 2 separate ones for "Central-Login" to demonstrate both options. As part of another ticket I will add a generic one for React

Comment (george-bafaloukas-forgerock) [reply javascript/central-login-pingone/README.md:20]: I deleted the Forgeorck / Ping references from the specific readme files

**Review by spetrov — APPROVED** [review]

**Review by spetrov — APPROVED** [review]
I reviewed the iOS and Android apps and did a quick sanity check of all apps/projects. 
Everything looks good to me, but here are a few things that we may want to consider in the future: 
- Improve the readme for the apps with easy to follow instructions - we currently have a link to the docs tutorial, which is correct only for the "java quick start" app (for Android)...
- We should probably remove the iOS `FRExample-ObjC` sample all together...
- It would be nice to have the iOS `FRAuthenticatorExample` in a separate project, instead of being part of the `FRExample` workspace.
- For consistency we should probably have the `iOS` directory with small letters?

File: iOS/README.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+To try out the Ping iOS SDK please look at one of our samples:
+
+- [**FR Example - `/uikit-frexamples/FRExample`**](./uikit-frexamples/)
+  
+  - An example iOS project making full use of the Ping SDK with a prototyping UI. Allows to explore the SDK uses in detail with example calls for running Journeys, getting OAuth2.0 tokens, User Info and more. Running the workspace makes use of the `FRExample`, `FRExampleObjC` and `FRAuthenticatorExample` projects.
+  (https://docs.pingidentity.com/sdks/latest/sdks/tutorials/ios/index.html)
+
+- [**Passkeys Sample - `/uikit-passkeys-sample`**](./uikit-passkeys-sample/UnsummitAuthentication)
```
Comment (spetrov) [inline iOS/README.md:21]: Broken link - should be: ./uikit-passkeys/UnsummitAuthentication)

File: android/kotlin-ui-prototype/README.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+  <hr/>
+</p>
+
+## Kotlin UI Prototype
+
+Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
+
+### Integrate with PingOne Advanced Identity Cloud / PingAM:
+
+- An example Android project making full use of the Ping SDK with a prototyping UI. Allows to explore the SDK uses in detail with example calls for running Journeys, getting OAuth2.0 tokens, User Info and more.
```
[the remark this answers is not in the document]
Comment (george-bafaloukas-forgerock) [reply android/kotlin-ui-prototype/README.md:14]: Good point thanks!

File: android/README.md (hunk 4 above)
Comment (forgerock-chris) [reply android/README.md:14]: Yeah is it's still up in the air I suggest writing around it, so you don't have to refer to it by a name, there's really no need that often.

So for example here we could just write:

"The following Android samples are available in this repo:"

No need to mention a name at all. Where we do, just "SDK" works, whatever the name becomes.

**Review by cerebrl — APPROVED** [review]
This looks good. I have a few comments for long term:

1. I'd like to use "web" instead of "javascript". It's like how we use Android, rather than Kotlin; or iOS, rather than Swift.
2. I think we need to rework the central login app to be server agnostic. Then, we can call it "oidc", rather than "aic"
3. We need to come up with what we're calling the apps for davinci and aic. Should it be `*-davinci` and `*-journey`, or `*-pingam` and `*pingone` ... ?

**Comment by george-bafaloukas-forgerock** [comment]
> * I'd like to use "web" instead of "javascript". It's like how we use Android, rather than Kotlin; or iOS, rather than Swift.
> * I think we need to rework the central login app to be server agnostic. Then, we can call it "oidc", rather than "aic"
> * We need to come up with what we're calling the apps for davinci and aic. Should it be `*-davinci` and `*-journey`, or `*-pingam` and `*pingone` ... ?

Captured as part of [SDKS-3539](https://pingidentity.atlassian.net/browse/SDKS-3539)
Let's refine this and we can schedule
